IOS@Taipei/IOS@Taipei/AppDelegate.swift

```diff
@@ -19,7 +19,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-        FirebaseApp.configure()
+        FirebaseApp.configure()        
```

Comment:
please don't add additional change if it doesn't mean anything.

Reply:
ok